components/TileCard/TileCard.tsx

```diff
@@ -22,46 +22,42 @@ const TileCardContent = ({
   superheadline,
   headline,
   link,
-  headingLevel = 'h3',
+  headingLevel,
 }: TileCardContentProps) => {
   const isExternalLink = link?.linktype !== 'story';
 
   return (
     <>
       {superheadline && (
-        <Text
+        <Heading
+          as={headingLevel}
           uppercase
+          font="sans"
           weight="semibold"
           tracking="wider"
           color={isDarkText ? 'black' : 'white'}
           className={styles.superhead}
         >
           {superheadline}
-        </Text>
+        </Heading>
```

Comment:
For the tile cards, put the heading in the superheading instead of in the larger "headline" below the superheading, because there is no content after the "headline"